### DISCUSSION TITLE
feature request: slack integration on github integration

### DISCUSSION THREAD

**Discussion by sourceful-karlson** [hasura/graphql-engine#8513] [General]
Git Deployment

Slack notification would be very benefitcial to our team

**Comment by meetzaveri** [comment]
Thanks for submitting a feature idea. 

We would recommend you to create a github issue on this with subject of feature request and follow it. That would be a good starting point and our team would be aware of it for tracking it in the future. You can easily do this via clicking [here](https://github.com/hasura/graphql-engine/issues/new?created_from_discussion_number=8513) or clicking "Create issue from discussion" which is present in the bottom of the right sidebar of this discussion page.
